### PR TITLE
9.0 Release

### PR DESCRIPTION
**Pull request by jennifer-shehane**
- [x] Changelog
- [x] Node.js changes
- [x] Remove Windows 32-bit support
- [x] 8.8.0 docs

### CODE REVIEW

File: content/_changelogs/9.0.0.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+- The `nodeVersion` configuration option has been deprecated and will be removed
+  in a future release.
+
+**Features:**
+
+- When `null` is passed as the encoding to `cy.readFile()`, the file is treated
+  as binary and read as a Buffer. Similarly, `null` passed as the encoding to
+  `cy.writeFile()` allows direct writing of buffers. If the encoding is
+  unspecified, the default remains `utf8`, matching the current behavior.
+  Addresses [#18534](https://github.com/cypress-io/cypress/issues/18534).
```
Comment (CypressCecelia) [inline content/_changelogs/9.0.0.md:46]: Should we also call out that this impacts fixture?

Based on: https://github.com/cypress-io/cypress-documentation/pull/4172

Comment (jennifer-shehane) [reply content/_changelogs/9.0.0.md:46]: Yes! Thanks.

Comment (chrisbreiding) [reply content/_changelogs/9.0.0.md:46]: This still needs to be addressed, right? What should be added concerning fixtures?

Comment (jennifer-shehane) [reply content/_changelogs/9.0.0.md:46]: I thought that I changed this.

Comment (jennifer-shehane) [reply content/_changelogs/9.0.0.md]: my bad, committing now.

**Review by chrisbreiding — APPROVED** [review]

**Pull request merged**